Resources/Prototypes/Actions/spells.yml

```diff
@@ -0,0 +1,90 @@
+﻿- type: action
+  actionType: SpellFireball
+  icon: Interface/Actions/Spells/fireball.png
+  name: "Fireball"
+  filters:
+    - spells
+  description: "Puts the fire at your fingertips! And explosive power!"
+  behaviorType: TargetPoint
+  behavior: !type:ProjectileSpell
+   castmessage: ONI SOMA! 
+   spellprojectile: FireBallbullet
+   speed: 15
+   cooldown: 1
+   ignorecaster: true
+
+- type: action
+  actionType: SpellTraps
+  icon: Interface/Actions/Spells/trap.png
+  name: "Traps!"
+  filters:
+    - spells
+  description: "Snare your enemies with a magical trap! Semi-transperent."
+  behaviorType: TargetPoint
+  behavior: !type:ProjectileSpell
+   castmessage: CAVERE INSIDIAS!
+   spellprojectile: TrapsEntitiy
+   speed: 0
+   cooldown: 1
+   ignorecaster: false
+   
+- type: action
+  actionType: SpellCarp
+  icon: Mobs/Aliens/Carps/magic.rsi/icon.png
+  name: "Carps!"
+  filters:
+    - spells
+  description: "Open a portal into space dragon's lair to summon a carp imbued with magic!"
+  behaviorType: TargetPoint
+  behavior: !type:ProjectileSpell
+   castmessage: RI'FATA!
+   spellprojectile:  MagicarpMob_Content
+   range: 2
+   speed: 0
+   cooldown: 1
+   ignorecaster: true
```

Comment:
As above re-using "ProjectileSpell" for this is messy. Same with the trap.

Reply:
Why is it messy? In gameplay it works fine and it saves having to bloat the game with "summon this, but stationary" script